docs: Remove implication of 18.04 build support (#5562)
KhronosGroup/Vulkan-ValidationLayers: BUILD.md

### Linux Build Requirements

> This repository has been built and tested on the two most recent Ubuntu LTS versions.

Reviewer: > Users can still build on 18.04 however, they can't use the default packages the come with the distro. They would have to setup their own build environment.

This information could be added in some extent

Reviewer: ```suggestion
This repository is built and tested on the two most recent Ubuntu LTS versions.
```

to make it sound like we will on-going keep doing this

Author: > This information could be added in some extent

I can't think of way to add that information that wouldn't age badly or require a lot of explanation.
